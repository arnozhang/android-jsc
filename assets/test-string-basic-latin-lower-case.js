function test () {
  return 'L0WER THAN YOU'.toLowerCase()
}
