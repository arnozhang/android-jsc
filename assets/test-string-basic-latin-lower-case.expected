l0wer than you
